app/src/main/res/layout/dialog_response.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<HorizontalScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center_vertical">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:background="@color/color_background_default_Image"
+        android:orientation="vertical"
+
+        >
```

Comment:
remvoe the unnecessary spaces and finish the statement with 

android:orientation="vertical" >

Reply:
Done!